www/src/components/guidelines/containers.js

```diff
@@ -1,39 +1,65 @@
 /** @jsx jsx */
 import { jsx } from "theme-ui"
-import styled from "@emotion/styled"
-import propTypes from "@styled-system/prop-types"
+import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 
-import { Box } from "./system"
-
-export const Container = styled(Box)()
-
-Container.defaultProps = {
+const container = {
+  boxSizing: `border-box`,
   position: `relative`,
-  pr: { xxs: 6, md: 10 },
-  pl: { xxs: 6, md: 10 },
+  px: 6,
   zIndex: 1,
 }
 
-export const Section = styled(Container)()
-
-Section.propTypes = {
-  ...propTypes.space,
-}
+export const Container = ({ children, ...rest }) => (
+  <div
+    sx={{
+      ...rest,
+      ...container,
+      [mediaQueries.md]: {
+        px: 10,
+      },
+    }}
+  >
+    {children}
+  </div>
+)
 
-Section.defaultProps = {
-  ...Container.defaultProps,
-  as: `section`,
-  pt: { xxs: 4, sm: 5, md: 8 },
-  pb: { xxs: 4, sm: 5, md: 8 },
-}
+export const Section = ({ children, ...rest }) => (
+  <section
+    sx={{
+      ...rest,
+      ...container,
+      py: 4,
+      [mediaQueries.sm]: {
+        py: 5,
+      },
+      [mediaQueries.md]: {
+        py: 8,
```

Comment:
Did you try the "responsive array" `theme-ui` syntax here, and it didn't work? If you want to not set a value for an individual breakpoint, you can do e. g. `px: [ 2, null, null, 3 ]` IIRC.